### PR TITLE
8.0 l10n it website sale fiscalcode

### PR DESCRIPTION
**Pull request by hurrinico**
ADD 3 modules
l10n_it_website_sale_fiscalcode l10n_it_sale_associations l10n_it_website_sale_corrispettivi 
depends on https://github.com/OCA/l10n-italy/pull/230 and https://github.com/OCA/e-commerce/pull/108

### CODE REVIEW

File: l10n_it_website_sale_corrispettivi/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    def checkout_values(self, data=None):
+        res = super(WebsiteSale, self).checkout_values(data=data)
+        cr, uid, context, registry = (
+            request.cr, request.uid, request.context, request.registry)
+        orm_user = registry.get('res.users')
+        partner = orm_user.browse(
+            cr, SUPERUSER_ID, uid, context).partner_id
+        if partner.is_company:
+            res['checkout']['invoice_or_receipt'] = "invoice"
+        if partner.is_company:
```
Comment (tafaRU) [inline l10n_it_website_sale_corrispettivi/controllers/main.py]: I think this is a typo, isn'it?

**Comment by eLBati** [comment]
Thanks @hurrinico , please check lint
https://travis-ci.org/OCA/l10n-italy/jobs/141264142

**Comment by hurrinico** [comment]
@eLBati travis is now red because this pr depends on https://github.com/OCA/e-commerce/pull/108 and https://github.com/OCA/l10n-italy/pull/230

**Comment by paolovalier** [comment]
testing locally I found that opening a _Corrispettivo_ (Accounting -> Corrispettivo) I can't find the button _Register Payment_ but if I go to the corresponding Sale Order and choose _View Invoice_ I get the same _Corrispettivo_ but with the button _Register Payment_
is this behavior intentional? I would deem useful to have the chance to register the payment from the _Corrispettivo_ document
two screenshot attached 
thanks
schermata del 2016-07-25 17-59-42
schermata del 2016-07-25 17-59-51

**Comment by OpenCode** [comment]
@hurrinico novità circa questa PR?

**Pull request closed without merging**